### PR TITLE
Fehrhart patch retractions

### PR DESCRIPTION
**Pull request by fehrhart**
Please check, had some troubles with testing!
#850

### CODE REVIEW

**Comment by fnielsen** [comment]
# It is a bit easier to overlook the changes in GitHub if there is a single commit rather than the multiple commits (though `git diff master` seems to display all changes fine.
# The headline "Retracted papers in this journal" seems to be wrong because it is for a publisher, - not a journal
# The Javascript name and HTML IDs should correspond (Javascript name should be lower case too)

File: scholia/app/templates/publisher.html [hunk 1]
```diff
@@ -142,6 +174,16 @@ <h3 id="As function of number of published works">As function of number of publi
 </div>
 
 
+<h2>Retractions</h2>
+
+<id="Retractions">Retracted papers in this journal</h2>
```
Comment (egonw) [inline scholia/app/templates/publisher.html]: Change to "Retracted papers in journals from this publisher"?

File: scholia/app/templates/publisher.html [hunk 2]
```diff
@@ -142,6 +174,16 @@ <h3 id="As function of number of published works">As function of number of publi
...
+
+<table class="table table-hover" id="retracted"></table>
+
+<div class="embed-responsive embed-responsive-4by3">
```
Comment (egonw) [inline scholia/app/templates/publisher.html]: @fehrhart, why are lines 183-185 with the `iframe` here? If I follow the URL, I just get the WDQS. What are you trying to do here?

**Comment by fehrhart** [comment]
Testing now works for me.

One problem, the query works only on journals but not on publishers. Do we have a template for (scientific) journals? I could not find something suitable in the template folder.

**Comment by fehrhart** [comment]
Working on a different patch in the correct template.

**Pull request closed without merging**